src/social_db/mod.rs

```diff
@@ -0,0 +1,28 @@
+use strum::{EnumDiscriminants, EnumIter, EnumMessage};
+
+mod permissions;
+mod storage_management;
+
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(context = crate::GlobalContext)]
+pub struct SocialDbManagement {
```

Comment:
Rename it to just `SocialDb` to match the subcommand to avoid confusion.

Reply:
Done